aws/aws.go

```diff
@@ -110,41 +91,29 @@ func MakeParams(action string) map[string]string {
 	return params
 }
 
-// Create a new AWS server to handle making requests
-func NewService(auth Auth, service ServiceInfo) (s *Service, err error) {
-	var signer Signer
-	switch service.Signer {
-	case V2Signature:
-		signer, err = NewV2Signer(auth, service)
-	// case V4Signature:
-	// 	signer, err = NewV4Signer(auth, service, Regions["eu-west-1"])
-	default:
-		err = fmt.Errorf("Unsupported signer for service")
-	}
-	if err != nil {
-		return
-	}
-	s = &Service{service: service, signer: signer}
-	return
+// Create a new AWS service to handle making requests
```

Comment:
A couple of places we don't follow the official good convention for commenting functions and in others we do. Small nit pick. http://blog.golang.org/godoc-documenting-go-code